src/apps/folders/crud.py

```diff
@@ -45,14 +45,6 @@ async def get_users_folder_in_workspace(
             workspace_applets_query.c.folder_id == FolderSchema.id,
             isouter=True,
         )
-        query = query.join(
-            FolderAppletSchema,
-            FolderAppletSchema.folder_id == FolderSchema.id,
-        )
-        query = query.join(
-            AppletSchema,
-            AppletSchema.id == FolderAppletSchema.applet_id,
-        )
         query = query.where(AppletSchema.is_deleted.is_(False))
         query = query.where(FolderSchema.workspace_id == workspace_id)
         query = query.order_by(FolderSchema.id.desc())
```

Comment:
Could you please write the generated query? For me it seems not right. E.g. bacause you removed join of AppletSchema, but line 48 has AppletSchema filter